### ISSUE TITLE
Dynamic environment variables undefined during SSR on Windows (*not* fixed by #5955)

### ISSUE THREAD

**Issue by aloker** [sveltejs/kit#6152]
### Describe the bug

I already mentioned it in https://github.com/sveltejs/kit/issues/5941#issuecomment-1221102524 but it might get lost.

On Windows, the `env`-export of $env/dynamic/private and $env/dynamic/public is always an empty object. 

This issue was present on Linux, too, but #5955 apparently fixed it there (available since next.418). It did not, however, fix it for Windows.

Related: #5941, #5955

### Reproduction

Repro https://github.com/validData/kit-env-issues
The repo should be rather self explanatory - the page uses private and public, dynamic and static environment variables. Static environment variables work fine both on Windows and Linux, dynamic variables only work fully on Linux. On Windows, all server side dynamic variables are missing. On the client side, they work fine (though not shown here - hydration is deactivated)

Running on WSL:
![image](https://user-images.githubusercontent.com/140714/185857553-714c3491-d724-4734-a86f-643d4fc429ae.png)

Running on Windows:
![image](https://user-images.githubusercontent.com/140714/185857660-c21b0001-9b23-4611-88b8-0a398ab27a0f.png)

### System Info

```shell
System:
    OS: Windows 10 10.0.19043
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 10.60 GB / 31.81 GB
  Binaries:
    Node: 16.15.1 - c:\tools\node\node.EXE
    Yarn: 1.22.19 - c:\tools\node\yarn.CMD
    npm: 8.13.2 - c:\tools\node\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (104.0.1293.63)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @sveltejs/adapter-auto: next => 1.0.0-next.65
    @sveltejs/kit: next => 1.0.0-next.428
    svelte: ^3.44.0 => 3.49.0
    vite: ^3.0.4 => 3.0.9
```

### Severity

blocking an upgrade

**Comment by tcc-sejohnson** [comment]
@aloker, #6018 might fix this. Any way you can check it out and see if it does? I have access to Windows, but it's not particularly well-set-up for development.

**Comment by tcc-sejohnson** [comment]
@aloker 

Even easier -- that PR has been merged. Upgrade to latest Kit and let me know if you still see the issue. If so, I'll go through the pain of setting up my Windows environment... 🤞🏻

**Comment by aloker** [comment]
No, sorry, #6018 did not fix it.

  System:
    OS: Windows 10 10.0.19043
    CPU: (16) x64 Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
    Memory: 17.09 GB / 31.81 GB
  Binaries:
    Node: 16.15.1 - c:\tools\node\node.EXE
    Yarn: 1.22.19 - c:\tools\node\yarn.CMD
    npm: 8.13.2 - c:\tools\node\npm.CMD
  Browsers:
    Edge: Spartan (44.19041.1266.0), Chromium (104.0.1293.63)
    Internet Explorer: 11.0.19041.1566
  npmPackages:
    @sveltejs/adapter-auto: ^1.0.0-next.65 => 1.0.0-next.65
    @sveltejs/kit: ^1.0.0-next.429 => 1.0.0-next.429
    svelte: ^3.49.0 => 3.49.0
    vite: ^3.0.9 => 3.0.9

I'm still seeing the same issues.

**Comment by Kraftwurm** [comment]
I have the same problem. Windows, and the **$env/dynamic/private** is always empty.
I'm on : @sveltejs/kit@1.0.0-next.431

**Comment by jesperordrup** [comment]
Same observations.

Im on 1.0.0-next.444

**Comment by tcc-sejohnson** [comment]
I'm not sure when this was fixed, but it's fixed. I'm on Windows 10. Pulled down the repro. Updated Kit to latest. All success. Closing unless someone can provide a newer repro -- I've tried several ways to get it to fail and I can't. 😄

**Comment by DoisKoh** [comment]
Thanks, I've confirmed that it's fixed for me as well on @sveltejs/kit@1.0.0-next.456.
